nova/virt/vmwareapi/vm_util.py

```diff
@@ -708,6 +709,13 @@ def _get_device_disk_type(device):
             return constants.DEFAULT_DISK_TYPE
 
 
+def get_cluster_property(session, prop_list, cluster):
+    cluster_config = session._call_method(
+        vutil, "get_object_property", cluster,
+        prop_list)
+
+    return cluster_config
```

Comment:
This function is now duplicated to `cluster_util`. I think you should remove it here.